app/models/rp/available_report.rb

```diff
@@ -45,6 +48,14 @@ def self.for_user(user)
       AvailableReport.from(available_report.create_table_alias(authorized_and_public, 'rp_available_reports'))
     end
     
+    def set_file_ext
```

Comment:
this code can be made better.. 
find the index of mime_type in MIME_TYPES & then get the element at that index from FILE_EXT